### ISSUE TITLE
Increase Allowance is confusing to me

### ISSUE THREAD

**Issue by lauraweindorf** [stakeordie/scrt-auction#25]
Haha I didn't really understand what it meant when I created the auction, but I did it because I was supposed to.

From what I can tell it just gets the tokens I'm selling and deposits them into the auction contract, like an escrow.

Maybe change the verbiage to something like that?

![image](https://user-images.githubusercontent.com/436905/105763851-36ee9980-5f24-11eb-802c-4e066c868b05.png)

I think that the average user of the Secret Auctions app doesn't really care that it's a factory contract that the tokens are being transferred to and locked ... kind of thinking in layman's terms here. This verbiage is something a developer would be interested in knowing though imho.

**Comment by cankisagun** [comment]
We can change the text as such:

Last step before creating the auction is to allow auction contract to access your tokens. By setting allowance, you will enable the application to automate transactions for you.
